### PR TITLE
Load-balance-simple updated for weave 1.1

### PR DESCRIPTION
**Pull request by abuehrle**
Need someone to review this please.  Recompiled sample app in Go, and then I created new docker images, retagged, and pushed them to the dockerhub. 

Also, updated scripts and text.   @pidster  @davkal @errordeveloper 

Do we need to come up with some conventions for pushing these images to DockerHub? i.e. what tags would you all like to see on an image. 

Here are the options: 

https://docs.docker.com/userguide/dockerimages/

Whoever has a few spare cycles to test out and review.  thanks.  

Also, bad branch name. Forgot to switch before I started working in here.

### CODE REVIEW

**Comment by abuehrle** [comment]
Question for @rade  Should I be highlighting the proxy rather than ssh'ing onto the VMs or a combination thereof?  Let me know what is the best practice here.

**Comment by rade** [comment]
I don't understand the question. Can you give an example of what you have in mind?

**Comment by abuehrle** [comment]
I meant should I install dockertoolbox on my mac and run the docker commands on the VMs from my local machine through the proxy.  I think I just answered my own question, since I realized this adds complexity and may not be not necessary.   

If you could assign someone to review this. Would be great.

File: weave-loadbalance/README.md [hunk 1]
```diff
@@ -1,30 +1,37 @@
...
 sidebarweight: 18
-
 ---
 
 {% include product-vars %}
 
-## What you will build ##
+## What You Will Build ##
+
+Weave provides a software network optimized for visualizing and communicating with apps distributed among Docker containers. Using tools and protocols that are familiar to you, Weave's network provides a way for you to communicate between containerized apps distributed across multiple networks or hosts more quickly and efficiently.
```
Comment (pidster) [inline weave-loadbalance/README.md]: I don't think this paragraph is necessary in a guide.

File: weave-loadbalance/README.md [hunk 2]
```diff
@@ -32,168 +39,161 @@ to demonstrate our use of round robin DNS.
 * [Vagrant & VirtualBox](/guides/about/vagrant.html)
 * [Git](http://git-scm.com/downloads)
 
-## What you will need to complete this guide ##
+## Before You Begin ##
```
Comment (pidster) [inline weave-loadbalance/README.md:40]: Yes, better.

File: weave-loadbalance/README.md (hunk 1 above)
Comment (abuehrle) [inline weave-loadbalance/README.md]: thnks @pidster I'll remove this.

**Comment by abuehrle** [comment]
I'm going to merge this, since the file that is online now, won't work at all if someone tries to use it.

**Pull request merged**